main.go

```diff
@@ -40,13 +40,17 @@ const indexHTML = `<!DOCTYPE html>
     const pre = document.createElement('pre');
     pre.innerText = await resp.text();
     document.body.appendChild(pre);
-    return;
+  } else {
+	const src = await resp.arrayBuffer();
+	const go = new Go();
+	const result = await WebAssembly.instantiate(src, go.importObject);
+	go.argv = {{.Argv}};
+	go.run(result.instance);
```

Comment:
Use space chars for indentation for consistency.

Reply:
Fixed.